### ISSUE TITLE
QEMU patch version 

### ISSUE THREAD

**Issue by wenhuizhang** [kata-containers/kata-containers#6167]
Hi, to whom it may concern:

I am trying to match the qemu patch version  against the qemu repo (https://github.com/wenhuizhang/qemu) I forked, it seems like while the repo I forked ranges from version 0 to 2.10, the patches mentioned in kata (https://github.com/wenhuizhang/kata-containers/blob/wenhui/tools/packaging/qemu/patches/tag_patches/7a800cf9496fddddf71b21a00991e0ec757a170a/0011-DAX-virtiofsd-Add-setup-remove-mapping-handlers-to-p.patch) ranges from 4 to 6, any hints on how to match these two version system please?

**Comment by liubin** [comment]
Don't know how and when you forked the repo, but you can sync tags using(for example):

```bash
git fetch upstream
git rebase upstream/master
git push
git push --tags

```

**Comment by wenhuizhang** [comment]
Hi @liubin Thanks so much , I am trying to download QEMU source code 6.2.0 and compile with the patch you provided, however some configuration does not work well on my side. Such as Config_TPM_TIS and config_PVPANIC are not recognized by QEMU 6.2.0

**Comment by liubin** [comment]
Are you trying according to [this guide](https://github.com/kata-containers/kata-containers/blob/c6b7f690402b1e8c9c2bc02a5cc696f3ae0706f4/docs/Developer-Guide.md#build-a-custom-qemu)?

We have some scripts to help build qemu under https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/static-build/qemu.

**Comment by wenhuizhang** [comment]
> Are you trying according to [this guide](https://github.com/kata-containers/kata-containers/blob/c6b7f690402b1e8c9c2bc02a5cc696f3ae0706f4/docs/Developer-Guide.md#build-a-custom-qemu)?
> 
> We have some scripts to help build qemu under https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/static-build/qemu.

Yup , I followed this guide, however tried on a 5.15 machine with 5.15 guest. 

It seems like some libc in qemu-6.2.0 are deprecated in libc of linux 5.15, such as `sigsetjump`.
